### PR TITLE
Update User Guide - Edit Command and Clear Command

### PR DESCRIPTION
**Pull request by rolandyuwy**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -93,7 +93,25 @@ Examples:
...
+### Editing a food item : `edit`
+
+Edits the details of an existing food item in the food inventory.
+
+Format: `edit INDEX [d/DESCRIPTION] [p/PRIORITY] [e/EXPIRY DATE] [t/TAG]...`
+
+* Edits the food item at the specified `INDEX`. 
+* The index refers to the index number shown in the displayed food item list. 
+* The index **must be a positive integer** 1, 2, 3, …​
+* At least one of the fields must be provided.
```
Comment (bchenghi) [inline docs/UserGuide.md]: Very minor, but what do you think of "At least one of the optional fields must be provided."? Since INDEX is not optional.

**Review by bchenghi — COMMENTED** [review]
The changes look great! I left one comment in the review.

File: docs/UserGuide.md (hunk 1 above)
Comment (rolandyuwy) [reply docs/UserGuide.md]: Noted, thanks!

**Review by bchenghi — APPROVED** [review]
LGTM

**Pull request merged**